src/views/user/index.js

```diff
@@ -2,323 +2,46 @@
-
-                <DesktopSegment
-                  segmentLabel="creator"
-                  onClick={() => this.handleSegmentClick('creator')}
-                  selected={selectedView === 'creator'}
-                >
-                  Threads
-                </DesktopSegment>
-                <MobileSegment
-                  segmentLabel="search"
-                  onClick={() => this.handleSegmentClick('search')}
-                  selected={selectedView === 'search'}
-                >
-                  Search
-                </MobileSegment>
-                <MobileSegment
-                  segmentLabel="participant"
-                  onClick={() => this.handleSegmentClick('participant')}
-                  selected={selectedView === 'participant'}
-                >
-                  Replies
-                </MobileSegment>
-                <MobileSegment
-                  segmentLabel="creator"
-                  onClick={() => this.handleSegmentClick('creator')}
-                  selected={selectedView === 'creator'}
-                >
-                  Threads
-                </MobileSegment>
-              </SegmentedControl>
-
-              {hasThreads &&
-                (selectedView === 'creator' ||
-                  selectedView === 'participant') && (
-                  <Feed
-                    userId={user.id}
-                    username={username}
-                    viewContext={
-                      selectedView === 'participant'
-                        ? 'userProfileReplies'
-                        : 'userProfile'
-                    }
-                    hasNoThreads={this.hasNoThreads}
-                    hasThreads={this.hasThreads}
-                    kind={selectedView}
-                    id={user.id}
-                  />
-                )}
+    const { currentUser } = this.props;
 
-              {selectedView === 'search' && <Search user={user} />}
-
-              {!hasThreads && <NullState bg="null" heading={nullHeading} />}
-            </Content>
-            <Extras>
-              <ErrorBoundary fallbackComponent={null}>
-                <ColumnHeading>Member of</ColumnHeading>
-                <CommunityList
-                  currentUser={currentUser}
-                  user={user}
-                  id={user.id}
-                />
-              </ErrorBoundary>
-            </Extras>
-          </Grid>
-        </AppViewWrapper>
-      );
-    }
-
-    if (isLoading) {
-      return <LoadingScreen />;
-    }
-
-    if (hasError) {
-      return (
-        <AppViewWrapper>
-          <Titlebar
-            title={'User not found'}
-            provideBack={true}
-            backRoute={'/'}
-            noComposer
-          />
-          <ViewError
-            heading={'We ran into an error loading this user.'}
-            refresh
-          />
-        </AppViewWrapper>
-      );
+    if (currentUser && currentUser.id) {
+      return <CurrentUser />;
```

Comment:
Instead of rendering it at `/me/settings`, should we just redirect to the user? I.e. `/me/settings` redirects to `/users/mxstbr/settings` and `/me` redirects to `/users/mxstbr`?